recipes-devtools/python/python3-labgrid/environment

```diff
@@ -3,3 +3,8 @@
 
 LABGRID_COORDINATOR_IP=127.0.0.1
 LABGRID_COORDINATOR_PORT=20408
+
+### The coordinator service will listen to the following ip, if the service is also
+### enabled.
+
+LABGRID_COORDINATOR_LISTEN_IP=0.0.0.0
```

Comment:
Why can't we simply use `${LABGRID_COORDINATOR_IP}:${LABGRID_COORDINATOR_PORT}` from `/etc/labgrid/environment`?

Reply:
Because you might want to listen to 0.0.0.0 and connect to 127.0.0.1 and allow others to connect to your coordinator, even if you don't know its public IP address.

Reply:
That is where we differ in our goals. I want it be generic, because there is no reason to have it only fit your use case, It hinders people from using meta-labgrid on a generic coordinator. (Might not be the case for everyone, but why not?)
This allows us to have a machine dedicated as being a coordinator for other exporters.